guides/release/components/looping-through-lists.md

```diff
@@ -208,33 +174,16 @@ Next, let's add a way for the user to send a new message. First, we need to
 add an action for creating the new message. We'll add this to the
 `<NewMessageInput />` component:
 
-```handlebars {data-filename="app/components/new-message-input.hbs" data-diff="-1,+2,-3,+4"}
-<form>
-<form {{on "submit" this.createMessage}}>
-  <input>
-  <Input @value={{this.message}}>
-  <button type="submit">
-    Send
-  </button>
-</form>
-```
-
-We're using the `submit` event on the form itself here rather than adding a
-`click` event handler to the button since it is about submitting the form as a
-whole. We also updated the `input` tag to instead use the built in `<Input>`
-component, which automatically updates the value we pass to `@value`. Next,
-let's add the component class:
-
-```javascript {data-filename="app/components/new-message-input.js"}
+```gjs {data-filename="app/components/new-message-input.gjs"}
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
-import { action } from '@ember/object';
+import { on } from '@ember/modifier';
+import { Input } from '@ember/component';
```

Comment:
let's not re-introduce this.

We don't actually want to encourage use of `Input` as it breaks our whole one-way-data-flow paradigm, and I can't actually tell anyone how it works. It's literal black magic.